tutorials/timefreq/wavelets_tutorial.py

```diff
@@ -0,0 +1,166 @@
+# The scaling parameter is the scaling factor for the morlet transform algorithm.
+# The norm parameter is optional, and defines the normalization method for the algorithm.
+# It can be specified as 'sss', or as 'amp'.
+# Specifying 'sss' divides by the square root of the sum of the squares.
+# Specifying 'amp' divides by the sum of the amplitudes.
+#
+###################################################################################################
+
+# Settings for the wavelet transform Algorithm
+freqs=[1, 30]
+
+# Compute wavelet transform using compute morlet wavelet transform algorithm
+mwt = compute_wavelet_transform(sig, fs=500, freqs)
+
+###################################################################################################
+# Plot morlet wavelet transform
+# You can plot the wavelet-transformed data using matplotlib:
+
+import matplotlib.pyplot as plt
+plt.imshow(mwt)
+plt.show()
+
+
+###################################################################################################
+# Convolve Wavelet Algorithm
+# ----------------------------------
+#
+# Now, let's use the Convolve Wavelet algorithm to convolve a signal with a Complex
+# morlet wavelet. This is very similar to the morlet transform algorithm, but the
+# convolve wavelet algorithm returns a complex time series as an array.
+# The real part of this array is the filtered signal. You can also take np.abs()
+# of the array to get the analytic amplitude. Similarly, you can take np.angle()
+# of the returned array to get the analytic phase.
+#
+# The Convolve Wavelet algorithm takes parameters sig, fs, freqs, n_cycles,
+# scaling, wavelet_len, and norm.
+# The sig parameters is the time series signal.
+# The fs parameter is the sampling rate.
+# The freqs parameter defines the frequencies values or frequency range to estimate
+# with morlet wavelets.
+# The n_cycles parameter defines the number of cycles of the oscillation with the
+# specified frequency.
+# The scaling parameter is the scaling factor for the morlet wavelet.
+# The wavelet_len parameter defines the length of the wavelet. It can override the
+# freq and n_cycles input parameters if so desired.
+# The norm parameter is optional, and defines the normalization method for the algorithm.
+# It can be specified as 'sss', or as 'amp'.
+# Specifying 'sss' divides by the square root of the sum of the squares.
+# Specifying 'amp' divides by the sum of the amplitudes.
+#
+###################################################################################################
+
+# Settings for the convolve wavelet Algorithm
+freq=10
+
+# Convolve a signal with a complex morlet wavelet using convolve wavelet algorithm
+cts = convolve_wavelet(sig, fs=500, freq=10)
+
+###################################################################################################
+# Plot convolved wavelet
+
+# You can plot the filtered signal by plotting the real part of the convolved wavelet:
+plt.imshow(mwt_real)
```

Comment:
You never define `mwt_real` if this worked for you, you probably did some computations that you didn't save in the code